securedrop-proxy/debian/copyright

```diff
@@ -3,46 +3,5 @@ Upstream-Name: securedrop-proxy
 Source: https://github.com/freedomofpress/securedrop-proxy
 
 Files: *
-Copyright: 2018 Freedom of the Press Foundation <securedrop@freedom.press>
-License: GPLv3+
-
-Files: orderedmultidict*
```

Comment:
looking at the examples in https://www.debian.org/doc/packaging-manuals/copyright-format/1.0 it does appear that we should be pointing to the contact information and licenses of upstream dependencies. This is a lot of manual editing and we'd need to maintain it as we update dependencies. We could do something during the debian package build script, inspect `setup.py` in each dependency and add the contents of the `license`, `author`, and `author_email` to `copyright` since we have all the tarballs downloaded. Let's do that (or something like it in case someone else has a better idea) in a followup (not for the beta)?

Reply:
This is a good suggestion, but, we have to hope that all of our dependencies have license information in their `setup.py` files. A few years ago, [the story was different](https://anweshadas.in/a-walk-of-licenses-in-pypi/).